Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
```

Comment:
Lets also make changes to lib.rs to export the `authorizer.rs` module

Reply:
I've made the function public. The import path changed from `use clerk_rs::validators::actix::validate_jwt` to `use clerk_rs::validators::authorizer::validate_jwt`.